src/ButtonBase/ButtonBase.js

```diff
@@ -399,6 +404,7 @@ ButtonBase.defaultProps = {
   disableRipple: false,
   focusRipple: false,
   tabIndex: '0',
+  TouchRippleProps: {},
```

Comment:
Not needed.

Reply:
Ok, it is something I use to do in my projects and I couldn't see it it in Popover, I thought it might be an omission. Looks like spreading undefined is not a problem. :)

I guess I have to run a docs:build to fix the test_browser

Reply:
Yes, you need to run `yarn docs:api`.